packages/utils/src/sqs/peek-dlq-print-details.ts

```diff
@@ -0,0 +1,186 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { BadRequestError } from "@metriport/shared";
+import { SQS } from "aws-sdk";
+import fs from "fs";
+import { Config } from "../../../api/src/shared/config";
+
+/**
+ * This script saves the details of the messages in the DLQ to a file.
+ */
+
+// Link to the FHIR Server DLQ (https://sqs....)
+const fhirServerDlqUrl = "";
+const maxNumberOfMessagesPerQuery = 1;
+
+export type MessageDetails = {
+  messageId: string | undefined;
+  bucketName: string | undefined;
+  fileName: string;
+  cxId: string | undefined;
+  jobId: string | undefined;
+  startedAt: string;
+  patientId: string | undefined;
+};
+
+async function peekIntoQueue() {
+  const s3FileNames = new Set<string>();
+  if (!fhirServerDlqUrl) throw new BadRequestError("Missing FHIR Server DLQ URL");
+
+  const messageCount = await getMessageCountFromQueue(fhirServerDlqUrl);
+  console.log(`>>> Message count: ${messageCount}`);
+
+  const messageDetails: MessageDetails[] = [];
+  console.log(`>>> Getting messages from source queue...`);
+  for (let i = 0; i < messageCount; i++) {
+    const messagesOfRequest = await peekMessagesFromQueue(fhirServerDlqUrl, {
+      maxNumberOfMessagesPerQuery,
+      maxNumberOfMessages: maxNumberOfMessagesPerQuery,
+    });
+
+    if (!messagesOfRequest || !messagesOfRequest.length) {
+      console.log(`>>> No messages found`);
+      return { messageCount: 0, first10Items: [] };
+    }
+
+    messagesOfRequest.map(async m => {
+      const patientId = m.MessageAttributes?.patientId?.StringValue ?? "na";
+      const body = m.Body ? JSON.parse(m.Body) : undefined;
+      const s3FileName = body.s3FileName ?? "na";
+
+      const attr = m.MessageAttributes;
+      if (!attr) return;
```

Comment:
Wouldn't be better to check the msg attrib and then build them (cxId, jobId) along side patientId?

Also, we should prob warn/log if we're skipping the message, right?

Reply:
Woops, the co-pilot code got me again! 
It's funny cuz I had it disabled for a couple months, and I only turned it back on today and it instantly there's issues arising from it 😅

Reply:
Fixed